RecoDataProducts/src/ComboHit.cc

```diff
@@ -146,7 +146,7 @@ namespace mu2e {
           for(size_t ich=0;ich < size();++ich){
             ComboHit const& ch = (*this)[ich];
             for(auto iph : ch.indexArray())
-              shids[ich].insert(shids[ich].end(),pshids[iph].begin(),pshids[iph].end());
+              if(pshids.size() == 8) shids[ich].insert(shids[ich].end(),pshids[iph].begin(),pshids[iph].end());
```

Comment:
This effectively turns off filling these indices, as the # of hits is never(?) 8.   I suspect the real problem is how these indices are being interpreted in FlagBkgHits.  I suggest to revert this change and create an issue, specific to stereo hits.